roles/confluent.variables/defaults/main.yml

```diff
@@ -58,6 +58,18 @@ common_role_completed: false
 
 proxy_env: {}
 
+### The method of installation. Valid values are "package" or "archive". If "archive" is selected then services will not be installed via the use of yum or apt, but will instead be installed via expanding the target .tar.gz file from the Confluent archive into the path defined by `archive_destination_path`. Configuration files are also kept in this directory structure instead of `/etc`. SystemD service units are copied from the ardhive for each target service and overrides are created pointing at the new paths. The "package" installation method is the default behavior that utilizes yum/apt.
+installation_method: "package"
+
+### The Scala version of the Confluent Platform archive to download. Possible values: 2.11, 2.12, etc. If you don't have a specific version requirement then use the default.
+confluent_archive_scala_version: 2.12
+
+### The path the downloaded archive is expanded into. Using the default with a `confluent_package_version` of *5.5.1* results in the following installation path `/opt/confluent/confluent-5.5.1/` that contains directories such as `bin` and `share`, but may be overridden usinf the `binary_base_path` property.
+archive_destination_path: "/opt/confluent"
+
+### If the installation_method is 'archive' then this will be the base path for the configuration files, otherwise configuration files are in the default /etc locations. For example, configuration files may be placed in `/opt/confluent/etc` using this variable.
+config_base_path: "{{ archive_destination_path }}"
```

Comment:
I like what you did here, still including the if else in the other file, can the variable name be `archive_config_base_path` or something that describes is related to the archive installation

Reply:
Fixed in commit 1affe1cb